devicetypes/smartthings/fibaro-smoke-sensor.src/fibaro-smoke-sensor.groovy

```diff
@@ -54,25 +54,25 @@ metadata {
 		input description: "Enter the menu by press and hold B-button for 3 seconds. Once indicator glows WHITE, release the B-button. Visual indicator will start changing colours in sequence. Press B-button briefly when visual indicator glows GREEN",
 				title: "To check smoke detection state", displayDuringSetup: true, type: "paragraph", element: "paragraph"
 		input description: "Please consult Fibaro Smoke Sensor operating manual for advanced setting options. You can skip this configuration to use default settings",
-				title: "Advanced Configuration", displayDuringSetup: true, type: "paragraph", element: "paragraph"
-		input "smokeSensorSensitivity", "enum", title: "Smoke Sensor Sensitivity", options: ["High","Medium","Low"], defaultValue: "${smokeSensorSensitivity}", displayDuringSetup: true
-		input "zwaveNotificationStatus", "enum", title: "Notifications Status", options: ["disabled","casing opened","exceeding temperature threshold", "lack of Z-Wave range", "all notifications"],
+				title: "Advanced settings", displayDuringSetup: true, type: "paragraph", element: "paragraph"
+		input "smokeSensorSensitivity", "enum", title: "Smoke sensor sensitivity", options: ["High","Medium","Low"], defaultValue: "${smokeSensorSensitivity}", displayDuringSetup: true
+		input "zwaveNotificationStatus", "enum", title: "Notifications", options: ["None","Casing opened","Exceeding temperature threshold", "Lack of Z-Wave range", "All"],
```

Comment:
nit: add a space after each `,`

Reply:
We're going to have to do something to migrate all the existing enum selections to the updated options list. Otherwise the current selections won't be returned when the preferences are displayed. There are probably various ways this can be done so if you need ideas let's discuss offline.